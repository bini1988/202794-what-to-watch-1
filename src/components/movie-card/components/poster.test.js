import React from "react";
import renderer from "react-test-renderer";
import Poster from "./poster";
import card from "../../../mocks/movie-card";

it(`Poster correctly renders default markup`, () => {
  const tree = renderer.create(
      <Poster card={card}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
it(`Poster correctly renders small mode markup`, () => {
  const tree = renderer.create(
      <Poster card={card} mode="small"/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
it(`Poster correctly renders big mode markup`, () => {
  const tree = renderer.create(
      <Poster card={card} mode="big"/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
