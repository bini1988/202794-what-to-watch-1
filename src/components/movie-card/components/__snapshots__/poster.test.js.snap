// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poster correctly renders big mode markup 1`] = `
<div
  className="movie-card__poster movie-card__poster--big"
>
  <img
    alt="War of the Worlds"
    height="327"
    src="https://es31-server.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg"
    width="218"
  />
</div>
`;

exports[`Poster correctly renders default markup 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt="War of the Worlds"
    height="327"
    src="https://es31-server.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg"
    width="218"
  />
</div>
`;

exports[`Poster correctly renders small mode markup 1`] = `
<div
  className="movie-card__poster movie-card__poster--small"
>
  <img
    alt="War of the Worlds"
    height="327"
    src="https://es31-server.appspot.com/wtw/static/film/poster/War_of_the_Worlds.jpg"
    width="218"
  />
</div>
`;
